CSharp/Library/Microsoft.Bot.Builder/ConnectorEx/BotData.cs

```diff
@@ -65,7 +63,7 @@ public interface IBotDataStore<T>
         /// <param name="botStoreType"> The bot store type.</param>
         /// <param name="cancellationToken"> The cancellation token.</param>
         /// <returns>Bot record that is stored for this key, or "empty" bot record ready to be stored</returns>
-        Task<T> LoadAsync(IAddress key, BotStoreType botStoreType, CancellationToken cancellationToken);
+        Task<T> LoadAsync(IAddress key, BotStoreType botStoreType, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This change was for consistency. Many other places in the SDK we have this pattern; this unifies it in places partners were questioning the difference.